src/components/Result.tsx

```diff
@@ -38,6 +38,11 @@ const Result = () => {
               <th className={`${styles.th} ${styles.centered}`}>
                 {t('result.attempts')}
               </th>
+              {results.some((result) => result.question.referenceLink) ? (
```

Comment:
To improve readability, can you can think about extracting the condition in a variable.